www-static/client/log.js

```diff
@@ -372,7 +372,11 @@ CCC.Log.renderJson = function(json) {
       // {type: "narrate", source: "Max", where: "Hangout", text: "Max smiles."}
       // {type: "narrate", source: "Cat", where: "Hangout", text: "Cat meows."}
       var div = document.createElement('div');
-      div.appendChild(document.createTextNode(json.text));
+      var text = json.text;
+      if (json.source) {
+        text = json.source + ': ' + text;
```

Comment:
I still think this is not the right way to fix the problem—i.e., I think narrations should not in general be associated with objects.  (However, the change is correct in allowing attributed narrations to be displayed meaningfully.)